tests/test_login.py

```diff
@@ -0,0 +1,26 @@
+import pytest
+import time
+import requests
+import json
+
+from tests import conftest
+from pages.rptest import RpTest
+
+
+class TestLogin:
+
+    @pytest.mark.nondestructive
+    def test_login(self, selenium, ldap, token):
+        rpTest = RpTest(selenium)
```

Comment:
Its not really good to have vars with the same name, even if the case differs. Let's change ``rpTest`` to ``test``.